src/include/main/client_context.h

```diff
@@ -171,6 +178,8 @@ class ClientContext {
     std::unique_ptr<common::RandomEngine> randomEngine;
     // Attached database.
     Database* database;
+    // Default database.
+    AttachedKuzuDatabase* defaultDatabase;
```

Comment:
These two names `database` and `defaultDatabase` are confusing. It's hard to tell what's the difference.

In my understanding, `defaultDatabase` should be renamed to `attachedDatabase`, and `database` should be `defaultDatabase`. Or we can also call them `localDatabase` and `remoteDatabase`, as attach can only apply to httpfs for now.
Also update the comments if you do the renaming.